samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp

```diff
@@ -8,9 +8,9 @@
        <h:outputText value="#{generalMessages.assessment}" />
        <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
     </h:commandLink>
-<f:verbatim></span></li>
+<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
```

Comment:
> On Mar 31, 2016, at 12:09 PM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In samigo/samigo-app/src/webapp/jsf/section-activity/sectionActivityHeadings.jsp <https://github.com/sakaiproject/sakai/pull/2079#discussion_r58081366>:
> 
> > @@ -8,9 +8,9 @@
> >         <h:outputText value="#{generalMessages.assessment}" />
> >         <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.author.AuthorActionListener" />
> >      </h:commandLink>
> > -<f:verbatim></span></li>
> > +<f:verbatim rendered="#{authorization.adminTemplate and template.showAssessmentTypes}"></span></li>
> That’s close for the preceding item and open for the next.
> 
> It's closing a span and an li tag..... Why not place a whole logical section into a panelGroup that is controlled with a rendered attribute?
> 
> 
If I were writing it from scratch I probably would have done it that way. I think verbatim combining an end and a beginning is ugly. But I was trying to fix it, not rewrite it. Feel free to rewrite it.

> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/2079/files/a04e7f41bf3b0921b76db3da76cf541c2dd1e031#r58081366>